.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: java
+
+branches:
+  only:
+    - master
```

Comment:
why only run on master?

Reply:
running only on master based branches only. including master. so one can push as many commits as he likes and travis will be executed only when pr is raised